models/topics.model.js

```diff
@@ -19,6 +19,18 @@ exports.getArticleDataById = (id) => {
     .query("SELECT * FROM articles WHERE article_id = $1", [id])
     .then((result) => {
       if (result.rows[0]) return result.rows[0];
-      return Promise.reject({ status: 404, msg: 'article does not exist' });
+      return Promise.reject({ status: 404, msg: "article does not exist" });
+    });
+};
+
+exports.getCommentsDataByArticleId = (id) => {
+  return db
+    .query(
+      "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC",
+      [id]
+    )
+    .then((result) => {
+      if (result.rows.length > 0) return result.rows;
+      return checkExists("articles", "article_id", id);
```

Comment:
It would be better to invoke your models within your controller and use a Promise.all() to run them simultaneously. This separates the concerns of your controllers and model